cmd/run/run.go

```diff
@@ -0,0 +1,83 @@
+package run
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/odpf/meteor/config"
+	"github.com/odpf/meteor/core/recipe"
+	"github.com/odpf/meteor/logger"
+	"github.com/odpf/meteor/plugins"
+	"github.com/spf13/cobra"
+)
+
+// NewCmd creates a command object for the "run" action
+func NewCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "run [COMMAND]",
+		Short:   "Run meteor for provided receipes",
+		Example: "meteor run recipe.yaml",
+		Args:    cobra.ExactArgs(1),
+		Run: func(cmd *cobra.Command, args []string) {
+
+			fi, err := os.Stat(args[0])
+			if err != nil {
+				fmt.Println(err)
+				return
+			}
+			switch mode := fi.Mode(); {
+			case mode.IsDir():
+				rundir(args[0])
+			case mode.IsRegular():
+				run(args[0])
+			}
+		},
+	}
+
+	return cmd
+}
+
+// Run a single recipe
+func run(recipeFile string) {
+	c, err := config.LoadConfig()
+	if err != nil {
+		log.Fatal(err)
+	}
+	log := logger.New(c.LogLevel)
+	plugins.Log.Level = log.Level
```

Comment:
should we hide this Log level update behind a function?
e.g.
```
plugins.SetLogLevel(log.Level)
```

Reply:
or maybe we can even pass the logger to the plugins so we would be using the same instance.
e.g.
```
plugins.SetLog(log)
```

Reply:
Yes, we can do that. Also, we are using native logger and logrus both. Didn't change it this PR but we should use just one of them.

Reply:
oh thought it was changed this PR, tried my proposed changes in my local and had some issues due to how we register our plugins now.

let's keep it this way first then.